### PR TITLE
Add option to enable hang and crash dumps

### PR DESCRIPTION
**Pull request by nohwnd**
Adds options to enable crash and hang dumps from dotnet test commandline without using runsettings.

### CODE REVIEW

**Comment by nohwnd** [comment]
Waiting for this on vstest side to make it to sdk.
https://github.com/microsoft/vstest/pull/2438

**Review by ViktorHofer — APPROVED** [review]

**Comment by nohwnd** [comment]
@wli3 Looks like the pipeline is failing with the same on master, and in all other PRs that I could see. Is there an issue that tracks this build problem, please?

File: src/Cli/dotnet/commands/dotnet-test/LocalizableStrings.resx [hunk 1]
```diff
@@ -217,4 +226,39 @@ Outputs a 'Sequence.xml' file in the current directory that captures the order o
...
+This option is currently only supported on Windows, and requires procdump.exe and procdump64.exe to be available in PATH.
+Or PROCDUMP_PATH environment variable to be set, and point to a directory that contains procdump.exe and procdump64.exe. 
+The tools can be downloaded here: https://docs.microsoft.com/en-us/sysinternals/downloads/procdump 
+Implies --blame.</value>
+  </data>
+  <data name="CmdBlameCrashDumpTypeDescription" xml:space="preserve">
+    <value>The type of crash dump to be collected. Implies --blame-crash.</value>
+  </data>
+  <data name="CmdBlameHangDescription" xml:space="preserve">
+    <value>Run the tests in blame mode and enables collecting hang dump when test exceeds the given timeout. Implies --blame-hang.</value>
```
Comment (bgrainger) [inline src/Cli/dotnet/commands/dotnet-test/LocalizableStrings.resx:243]: This causes `dotnet test --help` to print:

> `--blame-hang                             Run the tests in blame mode and enables collecting hang dump when test exceeds the given timeout. Implies --blame-hang.`

This is self-referential and redundant. Should `Implies --blame-hang.` be part of the documentation for `CmdBlameHangTimeoutDescription`?

Comment (nohwnd) [reply src/Cli/dotnet/commands/dotnet-test/LocalizableStrings.resx:243]: That's a typo, should have been: Implies `--blame`

--blame is the option that observes which tests are running but does not create any dumps.

**Pull request merged**